x-pack/plugins/security_solution/public/cases/components/all_cases/index.tsx

```diff
@@ -245,10 +245,19 @@ export const AllCases = React.memo<AllCasesProps>(
             deleteCasesAction: toggleBulkDeleteModal,
             selectedCaseIds,
             updateCaseStatus: handleUpdateCaseStatus,
+            includeCollections:
+              selectedCases.length > 0 &&
+              selectedCases.some((caseObj: Case) => caseObj.type === CaseType.collection),
```

Comment:
I think is a good opportunity to move the `isCollection` helper from `all_cases/actions.tsx` to a helper file and export it.